coreservices/files/postgres/docker-entrypoint-initdb.d/vectors.sh

```diff
@@ -1,8 +1,11 @@
 #!/bin/bash
+
 set -euo pipefail # using the 'set -euo pipefail' in lieu of 'set -e' command to ensure that the script stops on the first error it encounters.
 
-psql -v ON_ERROR_STOP=1 --username "$POSTGRES_USER" --password "$POSTGRES_PASSWORD" --dbname "$POSTGRES_DB_NAME" <<-EOSQL
-    \i /docker-entrypoint-initdb.d/init.sql
+psql -v ON_ERROR_STOP=1 --username "$POSTGRES_USER" --dbname "$POSTGRES_DB" <<-EOSQL
+    CREATE EXTENSION IF NOT EXISTS vectors;
+    ALTER EXTENSION vectors UPDATE;
+    ALTER DATABASE "${POSTGRES_DB}" SET default_table_access_method = 'vectors';
 EOSQL
 
 if [ $? -eq 0 ]
```

Comment:
**Suggestion:** Remove the `if [ $? -eq 0 ]` check at the end of the script as it is redundant due to the `set -euo pipefail` command, which already ensures the script exits on the first error.
> 🔥  **High Priority** 
 > [category: **`maintainability`**, importance: **`8`**]
> **Reason:** Removing the redundant error check is a good suggestion because the `set -euo pipefail` command already ensures that the script will stop on the first error. This simplifies the script.
 

```suggestion
# No need for this check
```